async-clipboard/demo.css

```diff
@@ -0,0 +1,36 @@
+textarea {
+	display: block;
```

Comment:
whoaaaaaaaaa spacez

Reply:
there was no editorconfig 🐙

Reply:
yea, we should set that up :( and linting too.